docs/slack.rst

```diff
@@ -174,54 +174,130 @@ Our goal with Write the Docs is to be **the most welcoming community on the inte
-- **Only participate when you can be constructive** - Try to be constructive in all that you do, and otherwise consider not participating in a conversation.
-- **Say hi** - The world is a better place with a flood of hello's.
-- **Don't be snarky** - We are all trying the best we can.
-- **Be careful with jokes** - You know that you're joking, but some people might take you seriously.
-- **If you don't have anything nice to say, don't say anything at all**.
+In addition, we have several Slack-specific guidelines:
+
+Social tips
+~~~~~~~~~~~
+
+To have good experiences on Slack, create the conversations you want to be a part of.
+You can help us do this by following these tips:
+
+- **Say hi.**
+  Don't forget to introduce yourself in ``#intros``.
+  If you've been away a while, reintroduce yourself.
+  The world is a better place with a flood of hello's.
+- **Use inclusive language.**
+  Our community is multicultural, multiracial, multigendered, and generally diverse.
+  When posting in channels, avoid using words such as "guys" to address a mixed group; possible alternatives include "folks", "everyone", and "documentarians".
+- **Remember that we're all documentarians.**
+  The Write the Docs community is much more than a space for technical writers.
+  We welcome everyone who wants to learn more about creating documentation, including developers, testers, customer support agents, technical writers, program/product managers, developer relations specialists, and more.
+- **Only participate when you can be constructive.**
+  Try to be constructive in all that you do, and otherwise consider not participating in a conversation.
+- **Don't be snarky.**
+  We are all trying the best we can.
+- **Be careful with jokes.**
+  You know that you're joking, but some people might take you seriously.
+- **If you don't have anything nice to say, don't say anything at all.**
+
+We also recommend reading `Microaggressions: Making space for everybody <https://www.metafilter.com/microaggressions.mefi>`_, which goes over how to avoid small slights and hurtful behaviors, especially against members of marginalized groups.
 
 Online community issues
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 There is a specific set of behaviors that is prevalent in online communities that we'd like to address, to make sure our community doesn't fall into these traps. 
-
 Please avoid these behaviors:
 
-- **No Black Sheep** - There may be tools that you don't like. There is a good chance there is someone in the room who either helped create or likes to use that tool. Don't make them feel bad, they are likely just in a different context.
+- **No Black Sheep**. - There may be tools that you don't like.
```

Comment:
Or here (re don't diss developers)....

Reply:
Yeah, this is a good place to mention it. Revised with 65b5bc69.